internal/tracegen/config.go

```diff
@@ -47,6 +48,7 @@ func (c *Config) Flags(fs *flag.FlagSet) {
 	fs.StringVar(&c.Service, "service", "tracegen", "Service name prefix to use")
 	fs.IntVar(&c.Services, "services", 1, "Number of unique suffixes to add to service name when generating traces, e.g. tracegen-01 (but only one service per trace)")
 	fs.StringVar(&c.TraceExporter, "trace-exporter", "otlp-http", "Trace exporter (otlp/otlp-http|otlp-grpc|stdout). Exporters can be additionally configured via environment variables, see https://github.com/jaegertracing/jaeger/blob/main/cmd/tracegen/README.md")
+	fs.StringVar(&c.ExportToFile, "export-to-file", "", "Filename to write generated spans to (in JSON format) instead of sending to a collector")
```

Comment:
I was just thinking - adding this new option creates an ambiguity, which one takes precedence, this one or `trace-exporter`. We can keep it to a single flag if we allow `file:{filename}` syntax.

Btw the [README](https://github.com/jaegertracing/jaeger/blob/main/cmd/tracegen/README.md) incorrectly says the option name is `-exporter`, can you fix it in the same PR?